lib/std/c.zig

```diff
@@ -166,7 +166,7 @@ pub extern "c" fn sendto(
     dest_addr: ?*const sockaddr,
     addrlen: socklen_t,
 ) isize;
-pub extern "c" fn sendmsg(sockfd: fd_t, msg: *const std.x.os.Socket.Message, flags: c_int) isize;
```

Comment:
Not quite, the libc/kernel definitions are set in stone and we shall try to follow them instead of changing the parameter signedness case-by-case.
This also avoids the horrendous conditional casts that one needs when switching between the libc interface and the syscall wrappers on Linux.

Reply:
By following them, do you mean to keep `flags` as c_int?

Reply:
Yep, sprinkle casts here and there as needed.

Reply:
Ok! Updated the definitions and added casts where necessary; thanks!